extensions/amp-ima-video/amp-ima-video.md

```diff
@@ -0,0 +1,74 @@
+<!---
+Copyright 2017 The AMP HTML Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+      http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS-IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+
+# <a name="amp-ima-video"></a> `amp-ima-video`
+
+<table>
+  <tr>
+    <td width="40%"><strong>Description</strong></td>
+    <td>A video player with an integration of the
```

Comment:
Rewrite as:

Embeds a video player for instream video ads that are integrated with the <a href="https://developers.google.com/interactive-media-ads/docs/sdks/html5/">IMA SDK</a>.